test/check-machines-create

```diff
@@ -911,6 +928,17 @@ class TestMachinesCreate(VirtualMachinesCase):
             elif self.os_name:
                 b.click("#os-select-group > div button")
                 b.click("#os-select li:contains('{0}') button".format(self.os_name))
+                b.wait_attr("#os-select-group input", "value", self.os_name)
+                if (self.expected_os_name and
+                    self.expected_location and
+                    self.sourceType == 'url'):
+                    b.set_input_text("#source-url", self.expected_location)
```

Comment:
Why don't you use self.location. expected_ prefix is used for when the dialog autofills that input field, in this case self.expected_os_name. The fields that we set in the test manually should not use expected.

Reply:
OK, then change it from expected_location to locationSecondInput